llmfoundry/data/finetuning/dataloader.py

```diff
@@ -292,13 +292,16 @@ def _build_hf_dataset_from_remote(
     """
     supported_extensions = ['jsonl', 'csv', 'parquet']
     finetune_dir = os.path.join(
-        os.path.dirname(os.path.realpath(__file__)),
-        f'downloaded_finetuning_data/{cfg.dataset.split}')
+        os.path.dirname(
+            os.path.dirname(os.path.dirname(os.path.realpath(__file__)))),
+        f'downloaded_finetuning/{cfg.dataset.split}')
```

Comment:
```suggestion
        os.path.dirname(
            os.path.dirname(os.path.dirname(os.path.realpath(__file__)))),
            downloaded_finetuning,
            cfg.dataset.split')
```

Reply:
Add a comment or just make a descriptive var for the base path like:
```
llmfoundry_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
```

Reply:
done.